scripts/install_R_packages.R

```diff
@@ -13,4 +13,9 @@ dependencies = c("naturalsort")
 if (!all(dependencies %in% rownames(installed.packages()))) {
   install.packages(setdiff(dependencies, rownames(installed.packages())), repos="http://cran.cnr.Berkeley.edu")
 }
+
+hmmUrl = "http://cran.r-project.org/src/contrib/HMM_1.0.tar.gz"
```

Comment:
This is for testing only, right?  We are not going to be dependent on another R package, right?